src/moscot/problems/generic/_mixins.py

```diff
@@ -127,9 +127,9 @@ def push(
         Parameters
         ----------
         source
-            Source key in :attr:`solutions`.
+            Source key in `solutions`.
```

Comment:
Is there a reason for this change?

Reply:
This was an error, kind of.  The mixin has no `solutions`, only the problem itself does.